### PR TITLE
Drop OpenCV Manager from samples initialization

### PR DESCRIPTION
**Pull request by asmorkalov**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [x] The PR is proposed to the proper branch
- [ ] There is a reference to the original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

**Comment by asmorkalov** [comment]
@opencv-alalek @mshabunin please review.

File: samples/android/15-puzzle/src/org/opencv/samples/puzzle15/Puzzle15Activity.java [hunk 1]
```diff
@@ -33,32 +32,19 @@ public class Puzzle15Activity extends CameraActivity implements CvCameraViewList
...
-
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         getWindow().addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);
 
+        if (OpenCVLoader.initLocal()) {
+            Log.i(TAG, "OpenCV loaded successfully");
+        } else {
+            Log.d(TAG, "OpenCV initialization failed!");
```
Comment (opencv-alalek) [inline samples/android/15-puzzle/src/org/opencv/samples/puzzle15/Puzzle15Activity.java:43]: Log.e?

File: samples/android/camera-calibration/src/org/opencv/samples/cameracalibration/CameraCalibrationActivity.java [hunk 2]
```diff
@@ -75,6 +55,16 @@ public CameraCalibrationActivity() {
     public void onCreate(Bundle savedInstanceState) {
         Log.i(TAG, "called onCreate");
         super.onCreate(savedInstanceState);
+
+        if (OpenCVLoader.initLocal()) {
+            Log.i(TAG, "OpenCV loaded successfully");
+        }
+        else {
```
Comment (opencv-alalek) [inline samples/android/camera-calibration/src/org/opencv/samples/cameracalibration/CameraCalibrationActivity.java:62]: `} else {` ?

File: samples/android/camera-calibration/src/org/opencv/samples/cameracalibration/CameraCalibrationActivity.java [hunk 3]
```diff
@@ -96,13 +86,9 @@ public void onPause()
...
     {
         super.onResume();
-        if (!OpenCVLoader.initDebug()) {
-            Log.d(TAG, "Internal OpenCV library not found. Using OpenCV Manager for initialization");
-            OpenCVLoader.initAsync(OpenCVLoader.OPENCV_VERSION_3_0_0, this, mLoaderCallback);
-        } else {
-            Log.d(TAG, "OpenCV library found inside package. Using it!");
-            mLoaderCallback.onManagerConnected(LoaderCallbackInterface.SUCCESS);
-        }
+        mOpenCvCameraView.enableView();
```
Comment (opencv-alalek) [inline samples/android/camera-calibration/src/org/opencv/samples/cameracalibration/CameraCalibrationActivity.java:89]: samples/android/15-puzzle/src/org/opencv/samples/puzzle15/Puzzle15Activity.java has check here.

We need to define a unified variant of code.

File: samples/android/15-puzzle/src/org/opencv/samples/puzzle15/Puzzle15Activity.java (hunk 1 above)
Comment (asmorkalov) [reply samples/android/15-puzzle/src/org/opencv/samples/puzzle15/Puzzle15Activity.java]: Applied for all samples.

**Review by opencv-alalek — APPROVED** [review]
Patch looks good to me.
However no chance to verify that now (no configured Android environment).

**Review by AleksandrPanov — APPROVED** [review]
Check PR with this config:
image
no regressions
There is a problem with camera-calibration, but it will be reproduced in 4.x

**Pull request merged**